dependency-check-suppressions.xml

```diff
@@ -106,4 +106,10 @@
         <packageUrl regex="true">^pkg:maven/io\.netty/netty\-tcnative\-classes@.*$</packageUrl>
         <cpe>cpe:/a:netty:netty</cpe>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+     file name: jakarta.el-3.0.3.jar
```

Comment:
can you specifiy the package url:
e.g.
<packageUrl regex="true">^pkg:maven/org\.springframework\.retry/spring\-retry@1.3.[12]$</packageUrl>